x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -323,27 +323,41 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
 
   public start(core: CoreStart, plugins: StartPlugins) {
     const savedObjectsClient = new SavedObjectsClient(core.savedObjects.createInternalRepository());
-
+    const registerIngestCallback = plugins.fleet?.registerExternalCallback;
```

Comment:
when is this used?  I don't see where the callback is registered now

Reply:
its passed to the `endpointAppContextService.start()` method - see line 360 below (need to expand source here on diff)